networking_arista/ml2/arista_sec_gp.py

```diff
@@ -607,9 +646,11 @@ def create_acl(self, sg, security_group_ips=None, existing_acls=None):
 
         num_rules = {'ingress': len(cmds['ingress']) - 2, 'egress': len(cmds['egress']) - 2}
 
-        # Try consolidation
+        # Try consolidation (or poor mans version, list(set()) is too slow for large ACL lists)
         if 0 < self.max_rules < num_rules['ingress'] + num_rules['egress']:
             cmds = self._consolidate_cmds(cmds)
```

Comment:
This consolidation should always run. Having less rules, which are functional equivalent is always good.
We need in that case another consolidation, which combines rules to be more permissive than the user would expect.

E.g.  Allow ingress from default (192.168.0.1, 192.168.0.3, 192.168.0.5...., 192.168.0.201) -> (192.168.0.0/24)

Reply:
Ok :)